src/components/dateTimePicker/DateTimePicker.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+
+class DateTimePicker extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: "",
+      time: ""
+    };
+  }
+
+  updateDate = () => {};
+
+  render() {
+    const currentYear = new Date().getFullYear();
+    return (
+      <div className="date-time-picker">
+        <select id="year" onChange={this.updateDate()}>
+          <option value="">Year</option>
+          <option value={currentYear}>{currentYear}</option>
+          <option value={currentYear + 1}>{currentYear + 1}</option>
+          <option value={currentYear + 2}>{currentYear + 2}</option>
+          <option value={currentYear + 3}>{currentYear + 3}</option>
+          <option value={currentYear + 4}>{currentYear + 4}</option>
+        </select>
+        <select id="month">
+          <option value="">Month</option>
+          <option value="Jan">Jan</option>
+          <option value="Feb">Feb</option>
+          <option value="Mar">Mar</option>
+          <option value="Apr">Apr</option>
+          <option value="May">May</option>
+          <option value="Jun">Jun</option>
+          <option value="Jul">Jul</option>
+          <option value="Aug">Aug</option>
+          <option value="Sep">Sep</option>
+          <option value="Oct">Oct</option>
+          <option value="Nov">Nov</option>
+          <option value="Dec">Dec</option>
+        </select>
+        <select id="day">
+          <option value="">Day</option>
+          <option value="1">1</option>
+          <option value="2">2</option>
+          <option value="3">3</option>
+          <option value="4">4</option>
+          <option value="5">5</option>
+          <option value="6">6</option>
+          <option value="7">7</option>
+          <option value="8">8</option>
+          <option value="9">9</option>
+          <option value="10">10</option>
+          <option value="11">11</option>
+          <option value="12">12</option>
+          <option value="13">13</option>
+          <option value="14">14</option>
+          <option value="15">15</option>
+          <option value="16">16</option>
+          <option value="17">17</option>
+          <option value="18">18</option>
+          <option value="19">19</option>
+          <option value="20">20</option>
+          <option value="21">21</option>
+          <option value="22">22</option>
+          <option value="23">23</option>
+          <option value="24">24</option>
+          <option value="25">25</option>
+          <option value="26">26</option>
+          <option value="27">27</option>
+          <option value="28">28</option>
+          <option value="29">29</option>
+          <option value="30">30</option>
+          <option value="31">31</option>
+          <option value="32">32</option>
+        </select>
```

Comment:
Create array of those values and use `map` to generate the `<options> `. You can also use date object and its api to create dynamic date selector.